test/e2e/storage/csi_mock_volume.go

```diff
@@ -348,6 +379,148 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 		})
 	})
 
+	Context("CSI Volume expansion [Feature:ExpandCSIVolumes]", func() {
+		tests := []struct {
+			name                  string
+			nodeExpansionRequired bool
+			disableAttach         bool
+		}{
+			{
+				name:                  "should expand volume without restarting node if nodeExpansion=off",
+				nodeExpansionRequired: false,
+			},
+			{
+				name:                  "should expand volume by restarting pod if attach=on, nodeExpansion=on",
```

Comment:
also test cases for:
* attach=off, nodeExpansion=on
* driver doesn't support expansion at all

Reply:
added more e2es that cover this.